pkgs/tools/inputmethods/fcitx5/with-addons.nix

```
@@ -1,4 +1,6 @@
-{ symlinkJoin, makeWrapper, fcitx5, fcitx5-lua, fcitx5-configtool, fcitx5-qt, fcitx5-gtk, addons ? [] }:
+{ lib, symlinkJoin, makeWrapper, fcitx5, fcitx5-lua, fcitx5-configtool, fcitx5-qt, fcitx5-gtk, addons ? [ ] }:
```

```suggestion
{ lib, symlinkJoin, makeWrapper, fcitx5, fcitx5-configtool, fcitx5-qt, fcitx5-gtk, addons ? [ ] }:
```

fcitx5-lua is an optional addon. I don't suggest add it as a dependency directly.